sql/tidb-specific.md

```diff
@@ -16,25 +16,197 @@ If you need to set the global variable, run:
 
 ```set @@global.tidb_distsql_scan_concurrency = 10 ```
  
-### tidb_distsql_scan_concurrency
-Scope: SESSION | GLOBAL
-Default value: 10
-This variable is used to set the concurrency of the `scan` operation. Use a bigger value in OLAP scenarios, and a smaller value in OLTP scenarios. For OLAP scenarios, the maximum value cannot exceed the number of CPU cores of all the TiKV nodes.
+### tidb_snapshot
+
+- Scope: SESSION
+- Default value: ""
+- This variable is used to set the time point at which the data is read by the session. For example, when you set the variable to "2017-11-11 20:20:20", the current session reads the data of this moment.
+
+### tidb_import_data
+
+- Scope: SESSION | GLOBAl
+- Default value: 0
+- This variable indicates whether to import data from the dump file currently.
+- When the variable is set to 1, the unique index constraint is not checked to speed up importing.
```

Comment:
Is it clearer to change this sentence to the following version?

"To speed up importing, the unique index constraint is not checked when the variable is set to 1."